core/drivers/pl022_spi.c

```diff
@@ -356,6 +289,21 @@ static inline uint32_t pl022_calc_freq(struct pl022_data *pd,
 	return pd->clk_hz / (cpsdvr * (1 + scr));
 }
 
+static void pl022_control_cs(struct spi_chip *chip, enum gpio_level value)
+{
+	struct pl022_data *pd = container_of(chip, struct pl022_data, chip);
+
+	if (pd->b_cs_callback && pd->cs_data.cs_cb) {
+		pd->cs_data.cs_cb(value);
+	} else if (!pd->b_cs_callback) {
+		while (read8(pd->base + SSPSR) & SSPSR_BSY)
+			DMSG("pl022 still transceiving - do NOT set CS yet!");
```

Comment:
I fear a trace overflow when the situation occurs.
If you feel this message is useful, maybe you could trace the debug message only once and add a "no more busy" message once busy signal cleared.